docs/source/documentation.html.md.erb

```diff
@@ -713,6 +713,23 @@ This position is an array of two numbers, representing the lattitude and the lon
     </tbody>
     </table>
 
+### Backend
+
+You can also query the Places API from your backend, here is an example with our [PHP API Client](https://github.com/algolia/algoliasearch-client-php). It requires a registered [Places account](https://www.algolia.com/users/sign_up/places).
+
+```php
+<?php
+$appId = 'YOUR_PLACES_APP_ID';
+$apiKey = 'YOUR_PLACES_API_KEY';
+
+$client = new \AlgoliaSearch\Client($appId, $apiKey);
+$places = $client->initPlaces($appId, $apiKey);
+
+$data = json_encode(array('query' => 'Paris'));
+$result = $places->search($data);
```

Comment:
I think this has to be called like:
```php
$places->search($query);
```

No need for the json_encode.
See: https://github.com/algolia/algoliasearch-client-php/blob/master/src/AlgoliaSearch/Index.php#L507